osf/migrations/0124_update_social_data_format.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-25 20:39
+from __future__ import unicode_literals
+
+import logging
+import progressbar
+
+from django.db import migrations, connection
+
+
+logger = logging.getLogger(__name__)
+
+
+FIELDS_TO_MIGRATE = [
+    'github',
+    'linkedIn',
+    'twitter'
+]
+
+
+BATCH_SIZE = 1000
```

Comment:
Drive-by: Are syntax checks passing (`inv syntax`)? This variable appears to be unused.

Reply:
Ah, perhaps uppercased constants don't count as unused variables because it's assumed that they may be imported by other modules.

In any case, I think it can be removed.